docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/configure-TTL.md

```diff
@@ -0,0 +1,68 @@
+# Configure TTL
+
+With **Time to Live** or TTL, **Nebula Graph** provides the ability to delete data automatically from a graph space after a certain time period. After you set the TTL , **Nebula Graph** will automatically remove these data after the time period, since the time they were last modified. Time to live value is configured in seconds. When you configure TTL, the system will automatically delete the expired data based on the TTL value, without needing a delete operation that is explicitly issued by the console.
```

Comment:
After you set the TTL , **Nebula Graph** will automatically delete the data after the TTL is expired(based on the last modification of the data),

Reply:
without needing -> without